### PR TITLE
LPD-41116 Update of client extensions through the UI can randomly fail in clustered environments

### PR DESCRIPTION
**Pull request by izaera**
QA analysis: https://github.com/liferay-frontend/liferay-portal/pull/4535#issuecomment-2456437691
Sign-off: https://github.com/liferay-frontend/liferay-portal/pull/4535#pullrequestreview-2414722598 https://github.com/liferay-frontend/liferay-portal/pull/4535#pullrequestreview-2414927056

### CODE REVIEW

**Comment by izaera** [comment]
ci:test:relevant

**Review by bryceosterhaus — APPROVED** [review]
LGTM!

**Comment by izaera** [comment]
ci:report:59449690

**Comment by izaera** [comment]
ci:report:59445794

**Comment by izaera** [comment]
**QA analysis**:

- Playwright failures are unrelated.

File: modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/service/impl/ClientExtensionEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -308,9 +308,6 @@ public ClientExtensionEntry updateClientExtensionEntry(
 			clientExtensionEntry.getTypeSettings(),
 			clientExtensionEntry.getType());
 
-		clientExtensionEntryLocalService.undeployClientExtensionEntry(
-			clientExtensionEntry);
-
```
Comment (dsanz) [inline modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/service/impl/ClientExtensionEntryLocalServiceImpl.java:313]: This makes sense, follows the pattern "first persist, then deploy" we added [here](https://github.com/liferay/liferay-portal/commit/70e133fde4765b6b0edb50442decdb510ecea8dc)

**Review by dsanz — APPROVED** [review]
LGTM ;)

**Comment by izaera** [comment]
ci:forward

**Pull request closed without merging**